### ISSUE TITLE
Set from 

### ISSUE THREAD

**Issue by gigi** [yii2tech/selfupdate#4]
Hi!
Thank you for this extension. It saved us a lot of time while our developing processes strating. However we have a little issue sending email reports via Yandex smtp
Yandex rejects email if "from" field not equal to "username" field.

So could you please add smth like 'reportFrom' property to config?
e.g.
```
SelfUpdateController.php
...
public $reportFrom;
...
protected function reportResult($subjectPrefix)  {
...
$from = $this->reportFrom;
if  ($from === null)  {
    $userName = @exec('whoami');
    if (empty($userName)) {
        $userName = Inflector::slug(Yii::$app->name);
    }
    $hostName = $this->getHostName();
    $from = $userName . '@' . $hostName;
} 
...
```

or even move this to separated method getFrom()

Thank you!

**Comment by klimov-paul** [comment]
Resolved by commit 13f0f735a7a5ed8625825838c99c9712c329c7bd

**Comment by gigi** [comment]
Thanx! When it will be available on packagist?
